pkg/controller/servingcert/controller/secret_updating_controller.go

```diff
@@ -207,6 +207,13 @@ func (sc *serviceServingCertUpdateController) ensureSecretData(service *v1.Servi
 		}
 		return true, nil
 	}
+
+	// set the owning-component unless someone else has claimed it.
+	if !update && len(secretCopy.Annotations[api.OwningJiraComponent]) == 0 {
+		secretCopy.Annotations[api.OwningJiraComponent] = api.OwningJiraComponent
```

Comment:
Shouldn't it be `secretCopy.Annotations[apiannotations.OpenShiftComponent]` instead?